python/langsmith/client.py

```diff
@@ -1325,22 +1367,13 @@ def batch_ingest_runs(
         if not create and not update:
             return
         # transform and convert to dicts
-        create_dicts = [self._run_transform(run) for run in create or EMPTY_SEQ]
+        create_dicts = [
+            self._run_transform(run, copy=True) for run in create or EMPTY_SEQ
+        ]  # still copy create dicts because manipulated in create_by_id
```

Comment:
Is this just fixing a bug (since we hadn't been copying these before)

Reply:
great catch removing. Ended up replacing with serialization path